### PR TITLE
Fix gas estimation translation

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: translate.js [hunk 1]
```diff
@@ -60,7 +60,15 @@ function translateJsonCompilerOutput (output) {
...
     var gasEstimates = contractInput['gasEstimates'];
-
+    if (!gasEstimates['creation']) {
+      gasEstimates['creation'] = [null, null];
+    }
+    if (!gasEstimates['internal']) {
+      gasEstimates['internal'] = null;
+    }
+    if (!gasEstimates['external']) {
+      gasEstimates['external'] = null;
```
Comment (axic) [inline translate.js]: Why does it need to be set to `null`?

Comment (yann300) [reply translate.js]: cause this is how `translateGasEstimates` manage it

Comment (axic) [reply translate.js]: Oh, probably that shouldn't check for `null` but for `undefined`. Will need to double check.

**Comment by yann300** [comment]
@chriseth @axic can we merge this?

File: translate.js (hunk 1 above)
Comment (yann300) [reply translate.js]: `translateGasEstimates` check for null

**Comment by axic** [comment]
@yann300 can you check this now? The previous version was creating invalid output, e.g. `{ "internal": "infinite" }` whereas `internal` is expected to be a hashmap.

**Comment by yann300** [comment]
looks good too

**Pull request merged**